packages/daemon/src/pet-store.js

```diff
@@ -9,7 +9,7 @@ const { quote: q } = assert;
 
 const validIdPattern = /^[0-9a-f]{128}$/;
 const validFormulaPattern =
-  /^(?:host|pet-store|pet-inspector|(?:readable-blob-sha512|worker-id512|pet-store-id512|eval-id512|make-unconfined-id512|make-bundle-id512|host-id512|guest-id512):[0-9a-f]{128}|web-bundle:[0-9a-f]{32})$/;
+  /^(?:host|pet-store|pet-inspector|(?:readable-blob-sha512|worker-id512|pet-store-id512|eval-id512|make-unconfined-id512|make-bundle-id512|host-id512|guest-id512):[0-9a-f]{128}|(?:lookup|web-bundle):[0-9a-f]{32})$/;
```

Comment:
In the fullness of time, I expect all of these to have the same CID breadth. `web-bundle` has the shorter CID because of the subdomain length restriction, but we’re going to move away from subdomains #1921. This should be an `id-512` like the rest. We will also, in the fullness of time, remove the `-id-512` suffix from all these and then make CIDs _something else entirely_ but at least homogeneous.

Reply:
That makes sense. In that case, I'll just use all 64 bytes of the digest.